x-pack/plugins/screenshotting/server/browsers/chromium/driver_factory/index.ts

```diff
@@ -151,8 +159,10 @@ export class HeadlessChromiumDriverFactory {
 
       logger.debug(`Browser page driver created`);
 
+      let isClosed = false;
```

Comment:
We have a `page: puppeteer.Page` variable, which keeps its own state on whether the page is open: 
https://pptr.dev/#?product=Puppeteer&version=v10.2.0&show=api-pageisclosed